config.yaml

```diff
@@ -10,20 +10,23 @@ options:
     type: string
     description: URL of the custom API that can be used to fulfill the DNS-01 challenge
   httpreq_http_timeout:
-    type: int
-    description: API request timeout
+    type: string
```

Comment:
Does this type matter? I don't see any type conversions in the code.

Reply:
Yes, to my understanding the error in #66 states that when executing a command in the workload the environment variables need to be strings. We use strings in other lego charms.